.travis.yml

```diff
@@ -32,9 +32,10 @@ matrix:
         - docker run -d --name orion1 --link mongodb:mongodb -p 1026:1026 fiware/orion -dbhost mongodb
       before_script:
         - npm run lint -C ./validator
-        - node ./validator/validate.js -i common-schema.json -i geometry-schema.json -i specs/Weather/weather-schema.json -i specs/Alert/alert-schema.json -p specs -c true
       script:
-        - true
+        - travis_wait 10 npm test -C ./validator
```

Comment:
why we are waiting here? I would prefer not to have CI that depends on waiting for a certain amount of time

@caa06d9c any ideas?

Reply:
This isn't really a wait within the CI, it is the standard mechanism to avoid a Travis
[Time out](https://docs.travis-ci.com/user/common-build-problems/#build-times-out-because-no-output-was-received) when no output is received over a period of time.

Link checking can take time and we don't want CI to fail because it hasn't completed yet.
If the step is taking more than 10 minutes (with no output) then the CI build **will timeout** and error. 10 minutes should be enough for now.